### PR TITLE
feat: Add support for spacer widget

### LINKED ISSUE #301: feat: Add support for the spacer widget.
**Description**

Add support for the spacer widget.

**Requirements**

- [ ] Add Mirai parser for the spacer widget

**Additional Context**

https://api.flutter.dev/flutter/widgets/Spacer-class.html

### PR DESCRIPTION
**Pull request by cscoderr**
## Description
This PR Add support for spacer widget

## Related Issues

Closes #301 

## Type of Change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Code refactor
- [ ] Build configuration change
- [ ] Documentation
- [ ] Chore

### CODE REVIEW

File: packages/mirai/lib/src/parsers/mirai_mirai_spacer/mirai_mirai_spacer.dart [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+export 'package:mirai/src/parsers/mirai_mirai_spacer/mirai_mirai_spacer_parser.dart';
+
+part 'mirai_mirai_spacer.freezed.dart';
+part 'mirai_mirai_spacer.g.dart';
+
+@freezed
+class MiraiMiraiSpacer with _$MiraiMiraiSpacer {
```
Comment (vkmaxcooldude) [inline packages/mirai/lib/src/parsers/mirai_mirai_spacer/mirai_mirai_spacer.dart:9]: Would it be okay to name it `MiraiSpacer`?

Comment (cscoderr) [reply packages/mirai/lib/src/parsers/mirai_mirai_spacer/mirai_mirai_spacer.dart:9]: oh! yes it was an oversight i push the update now

Comment (cscoderr) [reply packages/mirai/lib/src/parsers/mirai_mirai_spacer/mirai_mirai_spacer.dart]: Done

**Review by divyanshub024 — APPROVED** [review]
Looks great!! Thank you @cscoderr 💯

File: packages/mirai/lib/src/parsers/mirai_mirai_spacer/mirai_mirai_spacer.dart (hunk 1 above)
Comment (divyanshub024) [reply packages/mirai/lib/src/parsers/mirai_mirai_spacer/mirai_mirai_spacer.dart]: Thanks a lot, @vkmaxcooldude for the review. 🥇

**Pull request merged**